middleware/markdown/metadata/metadata.go

```diff
@@ -0,0 +1,161 @@
+package metadata
+
+import (
+	"bufio"
+	"bytes"
+	"os"
+	"time"
+)
+
+var (
+	// Date format YYYY-MM-DD HH:MM:SS or YYYY-MM-DD
+	timeLayout = []string{
+		`2006-01-02 15:04:05`,
+		`2006-01-02`,
+	}
```

Comment:
Mholt hates international datetime formats. This is where the revolution is started by us, @weingart.